src/diffusers/loaders.py

```diff
@@ -1455,7 +1518,41 @@ def _maybe_map_sgm_blocks_to_diffusers(cls, state_dict, unet_config, delimiter="
         return new_state_dict
 
     @classmethod
-    def load_lora_into_unet(cls, state_dict, network_alphas, unet, low_cpu_mem_usage=None, _pipeline=None):
+    def _optionally_disable_offloading(cls, _pipeline):
```

Comment:
Could you tell me why this method might be needed?

Reply:
I exposed this method to simplify the code blocks below
In case LoRA are loaded with PEFT it seemed crucial to first remove the cpu offloading hooks from the model, then attach the lora layers to finally re-enabled the offloading thanks to the block `# Offload back`